bigbluebutton-html5/imports/api/note/server/publishers.js

```diff
@@ -0,0 +1,23 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Note from '/imports/api/note';
+
+function note(credentials) {
+  const { meetingId, requesterUserId, requesterToken } = credentials;
+
+  check(meetingId, String);
+  check(requesterUserId, String);
+  check(requesterToken, String);
+
+  Logger.info(`Publishing note for ${meetingId} ${requesterUserId} ${requesterToken}`);
```

Comment:
We don't seem to use the requesterUserId or requesterToken. Do we need to try to ensure that the user subscribing is currently in the meeting? We do something like it in Breakouts' publish.js